tests/encryption_/test_schema.py

```diff
@@ -1,14 +1,21 @@
 from django.db import connections
 from django.test import TestCase
 
-from . import models  # your encryption_ models file with Encrypted*Test classes
+from . import models
 
 
 class SchemaTests(TestCase):
+    databases = {"default", "encrypted"}
     maxDiff = None
 
     # Expected encrypted fields map per model
     expected_map = {
+        "PatientRecord": {
```

Comment:
Patient (the top-level model) should be tested rather than PatientRecord.

Reply:
0d0b246a7ad7d690e922779929d4fac417dfa88a